rules/windows/process_creation/proc_creation_win_powershell_snapins_hafnium.yml

```diff
@@ -18,7 +18,9 @@ logsource:
     product: windows
 detection:
     selection:
-        Image: '*\powershell.exe'
+        Image|endswith:
+            - '\powershell.exe'
+            - '\pwsh.exe'
         CommandLine: '*add-pssnapin microsoft.exchange.powershell.snapin*'
```

Comment:
`CommandLine: '*add-pssnapin microsoft.exchange.powershell.snapin*'`
is
`CommandLine|contains: 'add-pssnapin microsoft.exchange.powershell.snapin'`